src/Files.App/Utils/Storage/StorageItems/NativeStorageFile.cs

```diff
@@ -154,14 +154,14 @@ public static IAsyncOperation<BaseStorageFile> FromPathAsync(string path)
 
 		private static bool CheckAccess(string path)
 		{
-			using var hFile = NativeFileOperationsHelper.OpenFileForRead(path);
+			using var hFile = Win32Helper.OpenFileForRead(path);
 			return !hFile.IsInvalid;
 		}
 
 		private static bool IsNativePath(string path)
 		{
 			var isShortcut = FileExtensionHelpers.IsShortcutOrUrlFile(path);
-			var isAlternateStream = RegexHelpers.AlternateStream().IsMatch(path);
+			var isAlternateStream = System.Text.RegularExpressions.Regex.IsMatch(path, @"\w:\w");
```

Comment:
Why the `RegexHelpers` class was removed?

Reply:
Omg, must be pull problem. I'll revert them.

Reply:
I checked all the files, and I'm nearly sure it's just here that is messed